02-into_to_imaging.Rmd

```diff
@@ -0,0 +1,17 @@
+
+# Introduction to Imaging
+
+
+## Learning Objectives
+
+```{r, fig.alt = "", out.width = "100%", echo = FALSE}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1ME0NbcIBmnHJRhX3JJyCwJuuomkl_BjJp6lD5oD5WnU/edit#slide=id.gd422c5de97_0_10")
+```
+
+## What are general guidelines of using imaging data?
+
+## An overview of the imaging types
+
+### Radiology
+
+### Histology/Pathology
```

Comment:
Did you want this part "A very basic bird’s eye overview of common general analysis methods that will be discussed more in later chapters - standardization ect. can get into ML a bit" of the outline in this section?